README.md

```diff
@@ -218,7 +218,16 @@ go-licenses save <package> [package...] --save_path=<save_path>
 Checking for forbidden and unknown licenses usage:
 
 ```shell
-go-licenses check <package> [package...] 
+go-licenses check <package> [package...]
+```
+
+**Tip**: Usually you'll want to add
+
+* `...` to check all the packages in your repository
```

Comment:
make it clear add ... to package

Reply:
Hmm, but the repository root is not necessarily an importable package.

Reply:
not sure I understand your comment, as your linked doc ... means a pattern to match packages, root does not need to be a package.

Reply:
I thought you meant that I should change to "`...` to package (i.e., `github.com/google/go-licenses`) to check all the packages in your repository", but maybe I misunderstood your point.

Reply:
ohh, I was not clear. let me clarify. I think this sentence lacks some context. It does not tell where to add ...

so I am asking you to make the sentence more complete. Add description that ... is appended to the end of an import path to include all packages matching that pattern.

Reply:
Thank you for the clarification! Fixed in [1d973cb](https://github.com/google/go-licenses/pull/162/commits/1d973cb0c61f93bcbe53b33f41bae9f929aaffcb).